internal/k8s/controller.go

```diff
@@ -645,10 +649,27 @@ func (lbc *LoadBalancerController) Run() {
 	}
 
 	if lbc.spiffeCertFetcher != nil {
-		err := lbc.spiffeCertFetcher.Start(lbc.ctx, lbc.addInternalRouteServer)
+		_, _, err := lbc.spiffeCertFetcher.Start(lbc.ctx)
+		lbc.addInternalRouteServer()
 		if err != nil {
 			glog.Fatal(err)
 		}
+
+		// wait for initial bundle
+		cert := <-lbc.spiffeCertFetcher.CertCh
+		lbc.syncSVIDRotation(cert)
```

Comment:
The previous code in spiffe.go that waited for this had a timeout involved so we didn't hang forever. We should do the same here.